README.md

```diff
@@ -4,7 +4,7 @@ A Python package to evaluate your ArchivesSpace instance for DACS [single-level
 
 DACSspace utilizes the ArchivesSpace API and a default JSON schema to validate resources. The output is a CSV containing a list of invalid URIs with the following fields: validation status, error count, and explanation.
 
-DACSspace also allows users to specify a schema to validate against other than the default DACS single-level required schema, see [Usage](https://github.com/RockefellerArchiveCenter/DACSspace#usage) section for more information.
+DACSspace also allows users to specify a schema to validate against other than the default DACS single-level required schema, see [Usage](https://github.com/RockefellerArchiveCenter/DACSspace/edit/readme-edits/README.md#what-schema-to-validate-your-data-against) section for more information.
```

Comment:
I don't think this link will work for anyone who can't edit the repository. What about the previous link wasn't working?

Reply:
Ah, shoot. I just noticed this. My thought was to specifically link to the subsection in "Usage" that talks about specifying a schema.